core/infra/config/peer_service_with_json.cpp

```diff
@@ -29,15 +29,16 @@ limitations under the License.
 #include "peer_service_with_json.hpp"
 #include "config_format.hpp"
 
-using PeerServiceConfig = config::PeerServiceConfig;
+namespace config {
+
 using txbuilder::TransactionBuilder;
 using type_signatures::Update;
 using type_signatures::Add;
 using type_signatures::Remove;
 using type_signatures::Peer;
 using nlohmann::json;
 
-std::vector<peer::Node> PeerServiceConfig::peerList;
+Nodes PeerServiceConfig::peerList;
```

Comment:
I'm sorry but, context of `peer_service_with_json.hpp` has migrated to `core/service/peer_service.hpp`.